cbc-repeated-iv/src/cbc_game.rs

```diff
@@ -0,0 +1,575 @@
+    ▀▀▀▀▀▀▀▀▀▀   ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀    ▀         ▀         ▀     ▀▀▀▀▀▀▀▀▀"
+                    .to_string()
+            ),
+            _ => (
+                "What are you doing here, step-bro??? UwU".to_string(),
+                "Okay, here's some small secret crypto-tip for you:".to_string(),
+                "https://www.youtube.com/watch?v=dQw4w9WgXcQ".to_string()
+            )
+        }
+    }
+
+    // Creating beautified ascii table
+    // User must send strings (lines), length of each line and padding
+    // We need to use old (3 years ago last commit) crate, because new one
+    // Can't handle Colorized strings properly
+    fn create_table(
+        title: &str,
+        data: Vec<Vec<&String>>,
+        padding: usize,
+    ) -> anyhow::Result<String> {
+        // At first let's create and build custom styles for title and main table
+        let main_table_format = FormatBuilder::new()
+            .column_separator('║')
+            .borders('║')
+            .separators(&[LinePosition::Top], LineSeparator::new('═', '╦', '╔', '╗'))
+            .separators(
+                &[LinePosition::Intern],
+                LineSeparator::new('═', '╬', '╠', '╣'),
+            )
+            .separators(
+                &[LinePosition::Bottom],
+                LineSeparator::new('═', '╩', '╚', '╝'),
+            )
+            .padding(padding, padding)
+            .build();
+
+        let title_table_format = FormatBuilder::new()
+            .column_separator('║')
+            .borders('║')
+            .separators(&[LinePosition::Top], LineSeparator::new('═', '╦', '╔', '╗'))
+            .separators(
+                &[LinePosition::Intern],
+                LineSeparator::new('═', '╬', '╠', '╣'),
+            )
+            .padding(padding, padding)
+            .build();
+
+        // Then let's create title table
+        let mut title_table = Table::new();
+        title_table.set_format(title_table_format);
+        // And set it with content
+        title_table.add_row(Row::new(vec![Cell::new(title)]));
+
+        // Creating main table
+        let mut main_table = Table::new();
+        main_table.set_format(main_table_format);
+
+        // Filling data into main table
+        for row in data {
+            // Fill row with cells
+            let mut cells_vector = vec![];
+
+            for cell in row {
+                cells_vector.push(Cell::new(cell.as_str()));
+            }
+
+            // Create new row and add it into main table
+            main_table.add_row(Row::new(cells_vector));
+        }
+        // Get main table as string variable
+        let mut main_table_str = main_table.to_string();
+        // And remove 'new line char' at the end of the string
+        main_table_str.pop();
+
+        // If there is no title => return only main table
+        if title.is_empty() {
+            Ok(main_table_str)
+        }
+        // Otherwise we need to pair them
```

Comment:
Just a curiosity: have you tried to use recursive tables, (the example is at the end of [this](https://github.com/phsym/prettytable-rs#using-macros) section of pretty-table.rs)?  Maybe with some style magic it can be done easier. This is just an assumption :)

Reply:
No, I think I would be left with more headache and problems than with current logic.